core/class/to_s_spec.rb

```diff
@@ -0,0 +1,13 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+require File.expand_path('../fixtures/classes', __FILE__)
+
+describe "Class#to_s" do
+  it "works when name is defined" do
+    CoreClassSpecs::Record.to_s.should == "CoreClassSpecs::Record"
+  end
+  
+  it "works with an anonymous class" do
+    c = Class.new
+    c.to_s.should =~ /#<Class:0x[0-9a-f]+>/
+  end
+end
```

Comment:
These examples could be merged with the ones in `core/module/to_s_spec.rb` of course.